pyodide_build/cli/build.py

```diff
@@ -164,6 +182,19 @@ def main(
     compression_level: int = typer.Option(
         6, help="Compression level to use for the created zip file"
     ),
+    no_isolation: bool = typer.Option(
+        False,
+        "--no-isolation",
+        "-n",
+        help="Disable building the project in an isolated virtual environment. "
+        "Build dependencies must be installed separately when this option is used",
+    ),
+    skip_dependency_check: bool = typer.Option(
+        False,
+        "--skip-dependency-check",
+        "-x",
+        help="Do not check that build dependencies are installed",
```

Comment:
Could you please mention that it is no-op when it is not used with `--no-isolation` IIUC?

Reply:
Addressed in 6c5da50434a00862e3e7d732504d62780fc6c3f8.